core/src/main/java/org/mule/runtime/core/api/exception/Errors.java

```diff
@@ -113,6 +114,11 @@
     public static final String TIMEOUT_ERROR_IDENTIFIER = "TIMEOUT";
 
     /**
+     * Indicates one or more errors occurred during routing.
+     */
+    public static final String COMPOSITE_ROUTING_ERROR = "COMPOSITE_ROUTING";
```

Comment:
Though I think this name makes sense for the exception, I'm not sure it would be easy to understand for users.